packages/ckeditor5-engine/src/controller/datacontroller.js

```diff
@@ -114,6 +100,29 @@ export default class DataController {
 		 */
 		this.viewDocument = new ViewDocument( stylesProcessor );
 
+		/**
+		 * Styles processor used during the conversion.
+		 *
+		 * @readonly
+		 * @member {module:engine/view/stylesmap~StylesProcessor}
+		 */
+		this.stylesProcessor = stylesProcessor;
+
+		/**
+		 * Data processor used specifically for HTML conversion.
+		 *
+		 * @member {module:engine/dataprocessor/htmldataprocessor~HtmlDataProcessor} #htmlProcessor
+		 */
+		this.htmlProcessor = new HtmlDataProcessor( this.viewDocument );
+
+		/**
+		 * Data processor used during the conversion.
+		 * Same instance as {@link #htmlProcessor} by default. Can be replaced at run time to handle different format, e.g. XML or Markdown.
+		 *
+		 * @member {module:engine/dataprocessor/dataprocessor~DataProcessor} #processor
+		 */
+		this.processor = this.htmlProcessor;
```

Comment:
[**@Reinmar**](https://github.com/Reinmar), I went with initializing the `processor` by default with `htmlProcessor` and removing initialization from the builds. Is there any reason `processor` was not created this way previously?

Reply:
The reason was that the DC did not create any DP so it made no sense to create here once if it was to be overridden by e.g. Markdown's DP. 

Now, if we have `this.htmlProcessor` anyway defined above, it's ok to have `this.processor` set to the same instance.